pkg/file/copy.go

```diff
@@ -0,0 +1,55 @@
+package file
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/pkg/errors"
+)
+
+func Copy(src, dst string, bufferSize int64) error {
```

Comment:
Assuming that we'll only ever be copying a standard file, can we implement this more simply as:

```go
srcFile, err := os.Open(src)
if err != nil {
  return err
}

defer srcFile.Close()

dstFile, err := os.Create(dst)
if err != nil {
  return err
}

defer dstFile.Close()

if _, err := io.Copy(dstFile, srcFile); err != nil {
  return err
}
```

?

Reply:
```ruby
FileUtils.cp(src, dst)
```

:trollface:

Reply:
I feel yuck copying a file fully into memory! But yeah, we know these are small, and as you point out, regular files, so let's simplify it.

Reply:
That doesn't copy everything into memory? `io.Copy` buffers internally

Reply:
Yeah 32mb. I guess I'm old and remember the days when I had less ram than that in my whole PC!

Reply:
Dude, you're copying it into an on-memory filesystem :joy: